cmake/utils.cmake

```diff
@@ -105,3 +104,11 @@ macro(append_to_windows_path_list path_list path)
             "${append_to_windows_path_list_tmp__}")
     endif()
 endmacro()
+
+# Strip path from all files in a list
+macro(strip_paths out_list path_list)
+    foreach(item ${path_list})
```

Comment:
Better start with a `set(${out_list} "")` to be safe from surprises.

Reply:
Will do.